packages/sanity/src/router/route.ts

```diff
@@ -8,27 +8,87 @@ import {decodeParams, encodeParams} from './utils/paramsEncoding'
 /**
  * @public
  */
-export type RouteNodeOptions = {
+export interface RouteNodeOptions {
+  /**
+   * The path of the route node.
+   */
   path?: string
+  /**
+   * The children of the route node. See {@link RouteChildren}
+   */
   children?: RouteChildren
+  /**
+   * The transforms to apply to the route node. See {@link RouteTransform}
+   */
   transform?: {
     [key: string]: RouteTransform<any>
   }
+  /**
+   * The scope of the route node.
+   */
   scope?: string
 }
 
 /**
+ * Interface for the {@link route} object.
+ *
  * @public
  */
-export const route: {
+export interface RouteObject {
```

Comment:
This probably needs a better name. There is already an interface with name `Route`